### PR TITLE
#335 - Re-enable IE 9 / 10

### LINKED ISSUE #335: Re-enable IE 9 and 10
In https://github.com/skatejs/skatejs/pull/757 we've implemented https://github.com/skatejs/skatejs/issues/742 which adds IE 9 / 10 support. This adds back in those browsers since our other repositories pass in 9 and 10. We can selectively disable if we need to.

### PR DESCRIPTION
**Pull request by treshugart**
Fixes #335.

Skate will have added support for IE 9 and 10 once it integrates the new build version from this.

### CODE REVIEW

File: browserstack.browsers.js [hunk 1]
```diff
@@ -102,6 +102,18 @@ const browserstackBrowsers = {
   },
 
   // IE (IE 8/9/10 not supported)
```
Comment (jpnelson) [inline browserstack.browsers.js]: this comment is out of date then, right?

**Comment by jpnelson** [comment]
lgtm

File: browserstack.browsers.js (hunk 1 above)
Comment (treshugart) [reply browserstack.browsers.js]: Yep! Gotta love comments!

**Comment by jpnelson** [comment]
lgtm

**Pull request merged**